modules/nw-route-specific-annotations.adoc

```diff
@@ -16,55 +16,122 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections`
+|Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. 
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp`
+|Limits the number of concurrent TCP connections made through the same source IP address. It accepts a numeric value. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+| 
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-http`
+|Limits the rate at which a client with the same source IP address can make HTTP requests. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`
+|Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/timeout` 
+|Sets a server-side timeout for the route. (TimeUnits) 
+|`spec.tuningOptions.serverTimeout` 
+|
+
+|`haproxy.router.openshift.io/timeout-tunnel` 
+|This timeout applies to a tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. 
+|`spec.tuningOptions.TunnelTimeout`
+|
+
+|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` 
+|You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop.
+|
+|`ingress.operator.openshift.io/hard-stop-after`
+
+|`router.openshift.io/haproxy.health.check.interval`
+|Sets the interval for the back-end health checks. (TimeUnits) 
+|`spec.tuningOptions.healthCheckInterval` 
+|
+
 |`haproxy.router.openshift.io/ip_whitelist`
-| Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
+|Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped. 
+
+The maximum number of IP addresses and CIDR ranges directly visible in the `haproxy.config` file is 61. [^1^] 
```

Comment:
This is an implementation detail that doesn't make sense to mention here, especially since we're removing the environment variables (a similar implementation detail) from the documentation.

Reply:
Yeah, I think the sentence about `haproxy.config` can be removed.